src/lib/certmap/sss_certmap_int.h

```diff
@@ -229,4 +255,16 @@ int add_principal_to_san_list(TALLOC_CTX *mem_ctx, enum san_opt san_opt,
 
 int rdn_list_2_dn_str(TALLOC_CTX *mem_ctx, const char *conversion,
                       const char **rdn_list, char **result);
+
+int rdn_list_2_component(TALLOC_CTX *mem_ctx, const char *conversion,
```

Comment:
Brief comment with args description would be helpful.

Reply:
Added a new patch with descriptions.